test/PolyhedralGeometry/markov_basis.jl

```diff
@@ -0,0 +1,25 @@
+@testset "Markov basis" begin
+    M =  [-1  -1  1   1   1   0  0   0   -1; -1  0   0   0   0   2  0   2   -3;
+          -1  0   0   0   2   0  0   0   -1; -1  0   0   1   0   1  0   0   -1;
+          -1  0   0   2   0   0  -1  0   0; -1  0   1   0   0   0  1   0   -1;
+          -1  1   -1  0   1   1  0   0   -1; -1  1   0   0   0   0  0   1   -1;
+          -1  1   0   0   0   0  1   -1  0;  -1  1   0   1   -1  0  0   0   0;
+          -1  2   -1  0   0   0  0   0   0; 0   -1  0   0   0   2  0   1   -2;
+          0   -1  0   0   1   1  0   0   -1; 0   -1  0   1   0   1  0   -1  0;
+          0   -1  0   1   1   0  -1  0   0; 0   -1  1   0   0   0  0   1   -1;
+          0   -1  1   1   -1  0  0   0   0; 0   0   -1  0   0   2  0   0   -1;
+          0   0   0   -1  0   1  0   2   -2; 0   0   0   -1  0   1  1   0   -1;
+          0   0   0   -1  1   0  0   1   -1; 0   0   0   -1  1   0  1   -1  0;
+          0   0   0   0   -1  1  0   1   -1; 0   0   0   0   0   0  -1  2   -1;
+          0   1   -1  0   -1  1  0   0   0; 1   -1  0   0   -1  1  0   0   0;
+          1   0   -1  -1  0   1  0   0   0]
+
+    S = [homogenize(v, 1) for v in lattice_points(cube(2))]
+    T = transpose(matrix(ZZ, Vector{Vector{fmpz}}(S)))
```

Comment:
```suggestion
    T = matrix(ZZ, transpose(Oscar.homogenized_matrix(lattice_points(cube(2)), 1)))
```

_edit:_ that doesn't make sense since S is needed as well ...

Reply:
so here i wanted to also test on vector of vectors input

Reply:
haha yup